test/ref_ops_test.cpp

```diff
@@ -2524,6 +2524,82 @@ TEST_CASE(gather_test)
     }
 }
 
+TEST_CASE(gather_dyn_test0)
+{
+    {
+        // Dynamic data, static indices
+        migraphx::program p;
+        auto* mm = p.get_main_module();
+        std::vector<int> data(2 * 3);
+        std::iota(data.begin(), data.end(), 0);
+        migraphx::shape s{migraphx::shape::int32_type, {{2, 5, 0}, {3, 3, 0}}};
+
+        auto x = mm->add_parameter("x", s);
+        std::vector<int> indices{1, 2};
+
+        migraphx::shape s_ind{migraphx::shape::int32_type, {1, 2}};
+        auto ind = mm->add_parameter("indices", s_ind);
+        mm->add_instruction(migraphx::make_op("gather", {{"axis", 1}}), x, ind);
+
+        migraphx::shape sresult{migraphx::shape::int32_type, {{2, 5, 0}, {1, 1, 0}, {2, 2, 0}}};
+        EXPECT(p.get_output_shapes().back() == sresult);
+        p.compile(migraphx::ref::target{});
+
+        migraphx::shape input_fixed_shape0{migraphx::shape::int32_type, {2, 3}};
+        migraphx::shape input_indices{migraphx::shape::int32_type, {1, 2}};
+        migraphx::parameter_map params0;
```

Comment:
```suggestion
        migraphx::parameter_map params;
```
Remove suffix `0` in all places unless necessary.